frontend/src/features/authz/authzSlice.js

```diff
@@ -17,10 +17,10 @@ import { signAndBroadcast } from "../../utils/signing";
 
 const initialState = {
   grantsToMe: {
-    status: "idle",
-    errMsg: "",
-    grants: [],
-    pagination: {},
+    // status: "idle",
```

Comment:
Why this code is commented?

Reply:
We are adding status, errMsg, grants, and pagination for each chain in the reducer. So, these are not used now.
I will remove these comments.